.scala-steward.conf

```diff
@@ -0,0 +1,20 @@
+# We need to configure this via: https://github.com/guardian/scala-steward-public-repos
```

Comment:
Maybe it's worth separating this into its own pr?

Reply:
Thanks for the suggestion - I've made it its own PR here: #7